### PR TITLE
✨ adding adunity ad network

### PR DESCRIPTION
**Pull request by jonnyau**
Adding adunity ad network to the list of supported ad networks.

### CODE REVIEW

File: ads/adunity.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/adunity.js]: nit: 2018

File: ads/adunity.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<amp-ad 
+    type="adunity"
+    width="300" height="250"
+    data-au-account="xxx" data-au-site="xxx" data-au-section="" data-au-zone="xxx"
+    data-au-dual="true">
+    <div placeholder></div>
+    <div fallback></div>
+  </amp-ad>
+```
+
```
Comment (calebcordry) [inline ads/adunity.md]: Could you mark required attributes and optional attributes here?

File: ads/adunity.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/adunity.md]: nit: 2018

File: examples/ads.amp.html [hunk 4]
```diff
@@ -454,6 +455,16 @@ <h2>AdThrive 320x50 banner</h2>
       data-sizes="320x50">
   </amp-ad>
 
+  <h2>AdUnity (300x250 demo banner)</h2>
+  <amp-ad 
+    type="adunity"
+    width="300" height="250"
+    data-au-account="5057446857420" data-au-site="test.site" data-au-section="" data-au-zone="rectangle"
```
Comment (calebcordry) [inline examples/ads.amp.html]: can we move to separate lines? Also `data-au-section` without the `=""` should be idiomatic way?

File: examples/adunity.amp.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<!doctype html>
```
Comment (calebcordry) [inline examples/adunity.amp.html]: please remove this file. This folder is not for ad examples.

File: ads/adunity.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+## Example
+
+```html
+<amp-ad 
+    type="adunity"
+    width="300" height="250"
+    data-au-account="xxx" data-au-site="xxx" data-au-section="" data-au-zone="xxx"
+    data-au-dual="true">
+    <div placeholder></div>
```
Comment (calebcordry) [inline ads/adunity.md]: These `(placeholder, fallback)` are not specific to this ad and should be removed.

File: ads/adunity.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<!---
```
Comment (calebcordry) [inline ads/adunity.md]: Would you link to this `md` from here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/amp-ad.md#supported-ad-networks

**Comment by jonnyau** [comment]
Hi, thanks for helping me out.
I will do those changes.

File: examples/ads.amp.html (hunk 4 above)
Comment (jonnyau) [reply examples/ads.amp.html]: data-au-section it's a required parameter but can be empty, should i remove it and change the code to handle missing data-au-section?

Comment (jonnyau) [reply examples/ads.amp.html]: seems to work ok idiomatic way

File: ads/adunity.md (hunk 7 above)
Comment (jonnyau) [reply ads/adunity.md]: I've added our link into supported ad networks.

**Comment by jonnyau** [comment]
Is there anything else i should do? Thanks.

**Comment by Enriqe** [comment]
friendly ping @calebcordry

**Comment by calebcordry** [comment]
@jonnyau Sorry for the delay, this is looking good. Are you ready for this to be merged?

**Comment by jonnyau** [comment]
Yes, i'm ready!
Figer crossed!
Thanks.

**Comment by calebcordry** [comment]
Thanks for contributing!

**Comment by jonnyau** [comment]
Thank you!

**Comment by jonnyau** [comment]
How long until we are able to use our tags live?

**Comment by calebcordry** [comment]
This should go to opt-in this week and prod next week.

**Comment by jonnyau** [comment]
Great, thank you!

**Pull request merged**